modules/explorer.go

```diff
@@ -5,89 +5,77 @@ import (
 )
 
 const (
+	// ExplorerDir is the name of the directory that is typically used for the
+	// explorer.
 	ExplorerDir = "explorer"
 )
 
-// Used for the BlockInfo call
 type (
-	ExplorerBlockData struct {
-		ID        types.BlockID   // The id hash of the block
-		Timestamp types.Timestamp // The timestamp on the block
-		Target    types.Target    // The target the block was mined for
-		Size      uint64          // The size in bytes of the marshalled block
-	}
+	// ExplorerStatistics returns a bunch of instantaneous statistics about the
```

Comment:
I was thinking more of the calculus sense of the word. It means the statistics as they appear at that very moment.